### ISSUE TITLE
Magento 1.9.3.6 safe for use on production site?

### ISSUE THREAD

**Issue by vishy93** [progammer-rkt/Rkt_SbCache#8]
I am using the latest magento version of date: 16th November 2017, and am facing the static block caching issue, which your extension duly fixes. 

Now, I would like to apply the fix to my production site thus can we confirm if this extension is safe for usage on production sites ?

**Comment by rajeev-k-tomy** [comment]
@vishy93 yes, you can safely use this in production, since it doesnt do any core hacks and it wont create any issues.
Make sure you test this and works as intended in your development environment
